reana_server/complexity.py

```diff
@@ -11,8 +11,37 @@
 from typing import Dict, List, Tuple
 
 from reana_commons.job_utils import kubernetes_memory_to_bytes
+from reana_commons.errors import REANAKubernetesMemoryLimitExceeded
 
-from reana_server.config import REANA_KUBERNETES_JOBS_MEMORY_LIMIT
+from reana_server.config import (
+    REANA_KUBERNETES_JOBS_MEMORY_LIMIT,
+    REANA_KUBERNETES_JOBS_MAX_USER_MEMORY_LIMIT,
+    REANA_KUBERNETES_JOBS_MAX_USER_MEMORY_LIMIT_IN_BYTES,
+)
+
+
+def validate_job_memory_limits(
+    workflow_name: str, complexity: List[Tuple[int, float]]
```

Comment:
*minor:* I think, `workflow_name` is also unnecessary as it is obvious from the error message in `reana-client` what failed:

```
...
==> ERROR: Cannot start workflow q-1.1: 
The "kubernetes_memory_limit" provided in the workflow (q-1.1) exceeds the limit (8Gi).
```

Reply:
Agree, my intention was to have the workflow name in the logs for better traceability, but it's easy to get it there from the requests